ci/open_stack_plugin/disk_image_builder/elements/jenkins-slave/pre-install.d/20-vault-repo

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set_vault_repo_rhel_7_6(){
+echo "adding vault repo to RHEL 7.6"
+cat >/etc/yum.repos.d/vault.repo <<EOL
+[vault_repo]
+name=Vault Repository
+baseurl=http://download.devel.redhat.com/composes/nightly/RHEL-7.6-20180809.n.0/compose/Server/x86_64/os/
```

Comment:
We need to use the other URL for **latest** repo `http://download.devel.redhat.com/composes/nightly/latest-RHEL-7.6/compose/Server/x86_64/os/`